src/lib/components/svg-view/Label.svelte

```diff
@@ -16,14 +16,14 @@
 		d="M5 20 l215 0"
 		x1={parentPosition.x}
 		y1={parentPosition.y}
-		x2={position.x + parentPosition.x}
-		y2={position.y + parentPosition.y}
+		x2={position.x}
```

Comment:
why was the parrent pos offset removed (is this fix implemented elswhere now or just removed?)